Asgardia.Constitution.Proposal.txt

```diff
@@ -359,15 +358,15 @@ Article 32. Asgardia’s Head of State
  d. Ministry of Administrative Affairs Chairperson
  e. Ministry of Justice Chairperson
 § 9. The Head of State:
- a. appoints the Supreme Justice of the Judicial Court, the Chairperson of the National Audit Office, the Chairperson of the Council of Supreme Values, Asgardian ambassadors extraordinary and plenipotentiary to Earth nation states, and any other space nation states;
+ a. appoints the Supreme Justice of the Judicial Court, the Chairperson of the National Audit Office, the Chairperson of the Council of Supreme Values, and Asgardian ambassadors extraordinary and plenipotentiary to Earth nation states, and any other space nation states;

```

Comment:
The "Asgardian ambassadors..." portion of the sentence, including all the way to the period, is the last item in this list, correct? This requires an "and" before if this is the case. If not, I'm having trouble interpreting the true meaning here.